src/apps/DEL_DOT_VEC_2D-Hip.cpp

```diff
@@ -122,7 +122,7 @@ void DEL_DOT_VEC_2D::runHipVariantImpl(VariantID vid)
 
       const size_t grid_size = RAJA_DIVIDE_CEILING_INT(iend, block_size);
 
-      hipLaunchKernelGGL((lambda_hip_forall<block_size, decltype(deldotvec2d_lambda)>),
+      hipLaunchKernelGGL((lambda_hip_forall_1D<block_size, decltype(deldotvec2d_lambda)>),
```

Comment:
Should this update be in another PR to keep this focused on the Kernel? I try to keep cuda and hip in sync.

Reply:
I can revert it and/or move it to a new branch, it got pulled in from one of your review comments and I foresee adding a  lambda_hip_forall using 2D thread indexing and wanted to get in front of it.

Reply:
Let's not rename it here.